src/icons/XIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
```

Comment:
this is good, but lets make the filename consistent with the others - think we're doing kebab-case everywhere